Userland/Applications/Maps/main.cpp

```diff
@@ -22,9 +23,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     // Map widget
     MapWidget::Options options {};
-    options.center.latitude = 30.0;
-    options.center.longitude = 0.0;
-    options.zoom = 3;
+    options.center.latitude = Config::read_string("Maps"sv, "MapView"sv, "CenterLatitude"sv, "30"sv).to_double().release_value();
+    options.center.longitude = Config::read_string("Maps"sv, "MapView"sv, "CenterLongitude"sv, "0"sv).to_double().release_value();
+    options.zoom = Config::read_i32("Maps"sv, "MapView"sv, "Zoom"sv, 3);
```

Comment:
You can't guarantee that `to_double()` will succeed, because it's reading from a user-editable config file. Rather than `release_value()`, use `value_or()` with the same default.